docs/web/errors.mdx

```diff
@@ -170,6 +197,33 @@ try {
 }
 ```
 
+## Timeouts
+
+Similar to the `signal` option for cancellation, there is also the `timeoutMs` option, which
+allows you to specify a timeout, in milliseconds, for an individual call in instances
+where the request is hung up or not responding. The timeout value is specified in the same
+options object described above.
+
+When a timeout is reached before the request has been completed, a `ConnectError` with code `DeadlineExceeded` will
+be thrown.
+
+An example using a timeout of 3 seconds:
+
+```ts
+try {
+  await client.say({sentence: "Hello"}, { timeoutMs: 3000 });
+} catch (err) {
+  if (err instanceof ConnectError && err.code === Code.DeadlineExceeded) {
+    // handle the timeout error
+  }
+}
+```
+
+Timeouts in Connect are the equivalent of [deadlines](https://grpc.io/docs/guides/deadlines/) in gRPC. They both
+behave similarly in that they will throw a "deadline exceeded" error. Since Connect's defined error codes map to the
+same codes and semantics as gRPC, this allows for easy interop between the two protocols.
```

Comment:
The timeout value is sent to the server, and honored on the server (and typically also on the client). This allows for propagation of the timeout to upstream services. 

More details in https://github.com/connectrpc/connect-es/releases/tag/v0.9.0, https://github.com/connectrpc/connect-es/pull/635, and https://github.com/connectrpc/connect-es/pull/591.

Since this is the documentation for web, an example for propagation is out of place here, but maybe we can still provide more context on the matter, because the feature is quite different from a standard idle timeout.